assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -1668,5 +1668,32 @@ describe("DiversionPage", () => {
         ).toBeChecked()
       })
     })
+
+    test("when no route is selected, clicking the finish button shows error message", async () => {
+      const routes = routeFactory.buildList(2)
+
+      render(
+        <RoutesProvider routes={routes}>
+          <DiversionPage
+            originalRoute={{
+              routePattern: undefined,
+              route: undefined,
+            }}
+          />
+        </RoutesProvider>
+      )
+
+      expect(
+        screen.getByRole("combobox", { name: "Choose route" })
+      ).not.toHaveAccessibleErrorMessage()
```

Comment:
note: this is here because I originally had a test which didn't assert this, and silently succeeded because it was always "visible". I then asserted `not.toBeVisible()` on the error content, which actually failed the test because we don't have access to the CSS in jsdom at this point. So this assertion makes sure it's known that there isn't an error _until_ the button is clicked.

Hmm, should this be a comment in the code?

Reply:
It wouldn't hurt to keep that comment around!

Reply:
Thanks! Done! https://github.com/mbta/skate/compare/d35f2b2cd0934ed079f0bce75ff9b3d7d4b04ec2..ee0a7305856cca85e6b6ec0161ce34cf18b1a981